frontend/app/src/components/profitloss/ReportMissingPrices.vue

```diff
@@ -101,6 +140,10 @@ export default defineComponent({
       await fetchHistoricalPrices();
     });
 
+    const refreshedHistoricalPrices: Ref<{ [key: string]: BigNumber }> = ref(
```

Comment:
nitpick: Record<string, BigNumber> in general I think it makes more sense to use TS standard library when possible